### DISCUSSION TITLE
How to use mergerfs if I want to one member fs to be read only

### DISCUSSION THREAD

**Discussion by g0r3** [trapexit/mergerfs#1152] [General]
Hi,

I have a remote file system, that is mounted on a local filesystem to `/remote`. The remote file systems sometimes contains archives, that I want to extract on the local system into the containing folder. But that would cause an upload of the extracted data to the remote filesystem. To avoid that, my idea was to create a `/local` directory and mount both folders with mergerfs to a folder `/merged`. The `/remote` folder will be read only and all extracted files are supposed to be created in `/merged`. How would I do that with mergerfs? I tried the NC and RO branch types, but this always fails to create a writeable filesystem in `/merged`. Only if I skip the `=NC` or `=RO` options, the created filesystem will be writeable. This is the command I used.

```
mergerfs -o allow_other,use_ino,cache.files=off,dropcacheonclose=true,category.create=all /local/:/remote=RO /merged
```

**Comment by g0r3** [comment]
Still hoping for an answer :-)

**Reply by trapexit** [reply]
The behaviors is as described in the docs. You set the mode of the branch to NC or RO if you don't want it considered for create calls. If you have other issues then you need to specifically described the details of the RW branch and general setup as detailed in the docs: https://github.com/trapexit/mergerfs#information-to-include-in-bug-reports

If /local isn't chosen then you have to look at the docs to see which condition is leading to it being skipped. 

Why do you have category.create=all? Just set it to `ff` or `mfs` or whatever is preferred for selection.

**Reply by g0r3** [reply]
I tried the command with `ff`, `mfs` and also with `all`. Latter was the last I've tried, that's why I ended up copying for the example command I've used here.

So you say that in principal my usage of mergerfs for the intended purpose is correct? Because I suspected that I made a mistake. Thats why I opened a thread here instead of filing a bug report. 

Thank you

**Reply by trapexit** [reply]
Yes, it looks fine. It's a widely used pattern.
